datajoint/fetch.py

```diff
@@ -199,15 +153,11 @@ def __getitem__(self, item):
         :return: tuple with an entry for each element of item
 
         Examples:
-
         >>> a, b = relation['a', 'b']
         >>> a, b, key = relation['a', 'b', datajoint.key]
-        >>> results = relation['a':'z']    # return attributes a-z as a tuple
```

Comment:
Attributes are defined by name, not by order.  It's one of the basic premises of the relational data model.  The order of the attributes does not matter.  Adding a new attribute to a relation should not break any code.  Besides, this feature can be accomplished by something like
```python
attrs = rel.heading.names
rel.fetch[*attrs[3:12]]
```